dbt_platform_helper/providers/platform_config_schema.py

```diff
@@ -275,6 +276,7 @@ def __opensearch_schema() -> dict:
                     Optional("ebs_volume_type"): str,
                     Optional("instance"): str,
                     Optional("instances"): int,
+                    # Explicitly setting 'master' is now deprecated and will need to be removed in the future. This property is now built-in to the OpenSearch plans.yml
```

Comment:
I don't think the `yml` suffix adds anything here, `OpenSearch plans` covers it.

You could include the URL https://platform.readme.trade.gov.uk/reference/configure-extensions/opensearch/plans/ though maybe.